### PR TITLE
Modify Texts according to PoEditor comments

### LINKED ISSUE #132: Modify texts
- [ ] Step 5 - Target Population
Description: "Insert the total population and age distribution (as a percent) for each health site where the campaign will be implemented. This data will be used to calculate coverage rates for the campaign. The source of data may be (whatever that analytic tool was - create a hyperlink if possible) or you may have access to local estimates based on population surveys through the Ministry of Health or other stakeholders that would be more updated or reliable. You may overwrite any existing data in HMIS, but please note that any changes you make in this step will only be applied once you run analytics."

- All comments at https://poeditor.com/projects/po_edit?order=comm_desc&filter=&id=229573&id_language=43&search=

### PR DESCRIPTION
**Pull request by matiasgarcia91**
### :pushpin: References

* **Issue:** Closes #132

### :memo: Implementation
Modified some texts according to poEditor comments. Some comments were written over outdated strings (not being used or already improved).

### CODE REVIEW

File: i18n/es.po [hunk 1]
```diff
@@ -140,22 +140,21 @@ msgid "Target Population"
...
-"por los antígenos seleccionados. La fuente de estos valores son las "
-"análiticas de DHIS2. De igual modo, cualquier cambio que haga en este paso, "
-"se aplicará únicamente cuando ejecute las analíticas."
 
 msgid ""
-"Specify target population, totals and age percentages for the age ranges "
-"required by all selected antigens."
+"Specify the target population and population distribution by age group "
+"(percent) for each antigen."
 msgstr ""
```
Comment (tokland) [inline i18n/es.po]: Are these two translations missing?

File: i18n/en.pot [hunk 2]
```diff
@@ -209,15 +211,6 @@ msgstr ""
...
 
-msgid "Population distribution (%)"
-msgstr ""
-
-msgid "Campaign Population Distribution"
-msgstr ""
-
-msgid "Missing"
-msgstr ""
-
```
Comment (tokland) [inline i18n/en.pot]: Those literals should not be removed. I think you need to do a "yarn link" our custom d2-i18n-extract. See:

https://github.com/EyeSeeTea/vaccination-app/issues/81#issuecomment-484438882

File: i18n/es.po (hunk 1 above)
Comment (matiasgarcia91) [reply i18n/es.po]: @adrianq Is going to fill them in

**Comment by adrianq** [comment]
I have merged and updated this PR with Spanish translations. I have created this issue https://github.com/EyeSeeTea/vaccination-app/issues/145 to take care of the missing hyperlink [here](https://github.com/EyeSeeTea/vaccination-app/pull/138/files#diff-f2b9f4b894da49f1eb21a35e9b332514R155)

**Pull request merged**